feature/login/src/main/java/com/iteneum/login/presentation/LoginView.kt

```diff
@@ -0,0 +1,200 @@
+package com.iteneum.login.presentation
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.border
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.text.ClickableText
+import androidx.compose.material3.*
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.text.AnnotatedString
+import androidx.constraintlayout.compose.ConstraintLayout
+import androidx.constraintlayout.compose.Dimension
+import com.iteneum.designsystem.components.LpFilledTonalButton
+import com.iteneum.designsystem.components.LpOutlinedTextFieldMail
+import com.iteneum.designsystem.components.LpOutlinedTextFieldPassword
+import com.iteneum.designsystem.R.drawable.*
+import com.iteneum.designsystem.theme.LeasePertTheme
+import com.iteneum.login.R
+
+/**
+ * Function that creates [LoginView] compose. This screen is used by the user to login on their account
+ * User will require to fill the correspondent information, such as email or password, or login
+ * with a social network
+ *
+ * @author Jesus Lopez
+ */
+@Composable
+fun LoginView() {
+    val sizes = LeasePertTheme.sizes
+
+    ConstraintLayout(
+        modifier = Modifier
+            .fillMaxSize()
+
+    ) {
+        val (logo, mail, password, tonalButton, loginWith, socialNetworkLogins, registerButton) = createRefs()
+
+        Image(
+            painter = painterResource(id = leasepert_logo),
+            contentDescription = stringResource(R.string.lv_logo),
+            modifier = Modifier
+                .size(sizes.extraSize142)
+                .constrainAs(logo) {
+                    top.linkTo(parent.top, margin = sizes.extraSize100)
+                    start.linkTo(parent.start)
+                    end.linkTo(parent.end)
+                }
+        )
+        LpOutlinedTextFieldMail(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(start = sizes.extraSize36, end = sizes.extraSize36)
+                .constrainAs(mail) {
+                    top.linkTo(
+                        logo.bottom,
+                        goneMargin = sizes.extraSize100,
+                        margin = sizes.extraSize100
+                    )
+                    start.linkTo(parent.start)
+                    end.linkTo(parent.end)
+                },
+            label = stringResource(R.string.lv_email),
+            isValid = false,
+            supportTextError = stringResource(R.string.lv_support_text_error),
+            onValueChange = { TODO() }
```

Comment:
Just comment out the TODO method so it doesn't crash the app for now like this `/*TODO("")*/` also adding a comment inside the todo of what needs to be done in the future would help other developers.
In general, we want this to apply to all our TODO's.